.github/CODEOWNERS

```diff
@@ -32,16 +31,13 @@ Cargo.toml
 # Engine (old)
 # This section should be removed once the engine moves to /app/engine
 /build.sbt @4e6 @jaroslavtulach @hubertp
-/distribution/ @4e6
+/distribution/ @4e6 @jdunkerley @radeusgd
 /engine/ @4e6 @jaroslavtulach @hubertp
 /project/ @4e6 @jaroslavtulach @hubertp
 /test/ @jdunkerley @radeusgd
-/tools/ @4e6
+/tools/ @4e6 @jaroslavtulach @radeusgd
 
 # Enso Libraries
 # This section should be amended once the engine moves to /app/engine
 /distribution/lib/ @jdunkerley @radeusgd
```

Comment:
I left this, as these are purely library files and so no sense in pinging @4e6 every time me and James modify a library file (almost every of our PRs).